package.json

```diff
@@ -52,10 +52,11 @@
     "@typescript-eslint/parser": "^2.34.0 || ^3.10.1 || ^4.0.0 || ^5.0.0",
     "aud": "^2.0.0",
     "babel-eslint": "^8 || ^9 || ^10.1.0",
-    "eslint": "^3 || ^4 || ^5 || ^6 || ^7 || ^8",
+    "eslint": "^8.18.0",
```

Comment:
This will break things; it needs to be reverted.

Reply:
Done. @ljharb